packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/media/LocalVideoTrack.kt

```diff
@@ -101,20 +97,28 @@ class CameraCapturer(
   }
 
   fun flipCamera() {
+    val enumerator = getEnumerator(context)
+    val wasFrontFacing = if (cameraName != null) enumerator.isFrontFacing(cameraName) else false
+
+    for (name in enumerator.deviceNames) {
+      if (wasFrontFacing && enumerator.isBackFacing(name) || !wasFrontFacing && enumerator.isFrontFacing(name)) {
+        cameraName = name
+        break
+      }
+    }
+    setMirrorVideo(!wasFrontFacing)
     cameraCapturer.switchCamera(this)
   }
 
   fun switchCamera(deviceName: String) {
+    val enumerator = getEnumerator(context)
+    cameraName = deviceName
+    setMirrorVideo(enumerator.isFrontFacing(deviceName))
```

Comment:
are you sure that the front camera is always mirrored? 
I know it's very common and normal but what if user decides otherwise?

Reply:
ok, correct me if I'm wrong but what if:
- user sets camera to be not mirrored
- switches from back camera to front camera
- setMirrorVideo is called with `true` because it's now a front camera and it will be mirrored but mirror property is still false?

Reply:
now there are two seperate variables, one for mirror and one for if front camera